pippo-core/src/main/java/ro/pippo/core/Application.java

```diff
@@ -248,7 +249,8 @@ public final RequestResponseFactory getRequestResponseFactory() {
      * @return
      */
     protected RequestResponseFactory createRequestResponseFactory() {
-        return new RequestResponseFactory(this);
+//        return new RequestResponseFactory(this);
+        return new GZipRequestResponseFactory(this);
```

Comment:
I think like @decebals , and hold `RequestResponseFactory` as it is.

I also consider that in the future you could create a wrapper or decorator to wrap several future implementations of `RequestResponseFactory`